services/order.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: datetime = None
+) -> Order:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
```

Comment:
Consider handling the `DoesNotExist` exception for `get_user_model().objects.get(username=username)` to avoid unhandled exceptions if a user with the given username does not exist. You can return `None` or raise a custom exception.